elys/chain.json

```diff
@@ -73,6 +73,10 @@
       "enabled": false
     }
   },
+"logo_URIs": {
+    "png": "https://github.com/cosmos/chain-registry/blob/master/elys/images/elys.png",
```

Comment:
The png url must point to the raw file.
https://raw.githubusercontent.com/cosmos/chain-registry/master/elys/images/elys.png

Reply:
Updated the link, thank you

Reply:
Sorry, but that's still not the right URL; it must go to the raw file directly (our URLs are not quite the same)